tests/TestHelpers/GraphHelper.php

```diff
@@ -285,6 +285,33 @@ public static function deleteUser(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param string $userId
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function deleteUserByUserId(
+		string $baseUrl,
+		string $xRequestId,
+		string $adminUser,
+		string $adminPassword,
+		string $userId
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'users/' . $userId);
+
+		return HttpRequestHelper::delete(
+			$url,
+			$xRequestId,
+			$adminUser,
+			$adminPassword,
+		);
+	}
+
```

Comment:
IMO, not needed for this PR
```suggestion
```

Reply:
i asked Viktor which method should we use he told me to use `user id` to delete the user. So i implemented deletion of user using `user-id`

Reply:
Was just trying to reduce the code changes if we can the existing helpers. But yeah, using ID seems to be the way to go as the clients use ids